src/controllers/controller.bar.js

```diff
@@ -184,33 +181,30 @@ function isFloatBar(custom) {
 
 class BarController extends DatasetController {
 
-	constructor(chart, datasetIndex) {
-		super(chart, datasetIndex);
-	}
-
 	/**
 	 * Overriding primitive data parsing since we support mixed primitive/array
 	 * data for float bars
 	 * @private
 	 */
-	_parsePrimitiveData() {
-		return parseArrayOrPrimitive.apply(this, arguments);
+	_parsePrimitiveData(...args) {
+		return parseArrayOrPrimitive.apply(this, args);
 	}
 
 	/**
 	 * Overriding array data parsing since we support mixed primitive/array
 	 * data for float bars
 	 * @private
 	 */
-	_parseArrayData() {
-		return parseArrayOrPrimitive.apply(this, arguments);
+	_parseArrayData(...args) {
+		return parseArrayOrPrimitive.apply(this, args);
 	}
 
 	/**
 	 * Overriding object data parsing since we support mixed primitive/array
 	 * value-scale data for float bars
 	 * @private
 	 */
+	// eslint-disable-next-line class-methods-use-this
```

Comment:
Can we turn off this check rather than having to disable? It looks like it's in 50+ places

Reply:
Yeah